docs/admin_tutorial.md

```diff
@@ -46,32 +48,389 @@ CREATE EXTENSION pg_diffix;
 ALTER DATABASE test_db SET session_preload_libraries TO 'pg_diffix';
 ```
 
-## Configuration
+## User configuration (per database)
 
-1\. Label the test data as personal (requiring anonymization):
+1\. Create a user account for the analyst:
+
+```
+CREATE USER analyst_role WITH PASSWORD 'some_password';
+```
+
+2\. Give the analyst read-only access to the test database:
+
+```
+GRANT CONNECT ON DATABASE test_db TO analyst_role;
+GRANT SELECT ON ALL TABLES IN SCHEMA public TO analyst_role;
+```
+
+3\. Label the analyst as restricted and trusted:
+
+```
+CALL diffix.mark_role('analyst_role', 'anonymized_trusted');
+```
+
+## Anonymization configuration (per table)
+
+1\. Label the test data table as personal (requiring anonymization):
 
 ```
 CALL diffix.mark_personal('test_table', 'id');
 ```
 
-2\. Create an account for the analyst:
+__That's it!__ The analyst can now connect to the database and issue (only) anonymizing queries against the test dataset. Refer to the [Administration Guide](admin_guide.md) for additional details, or if you wish to modify default anonymization parameters.
+
+# Real Examples
+
+The real examples are databases that are used for the Diffix bounty programs. The databases can be found at https://db001.gda-score.org/.
```

Comment:
To keep the tutorial short and not overwhelming, what do you think of moving this section to a separate file?